### DISCUSSION TITLE
BC Labels

### DISCUSSION THREAD

**Discussion by Jegelewicz** [ArctosDB/arctos#5909] [Ideas]
I was asked by a prospective collection if we support [BC Labels](https://localcontexts.org/labels/biocultural-labels/)

A resource for indigenous provenance that includes the [Local Contexts Hub](https://localcontextshub.org/login/)

This might be of interest to cultural collections.

@AJLinn @droberts49 @wellerjes @krgomez @marecaguthrie

**Comment by Jegelewicz** [comment]
@ArctosDB/diversity-and-inclusion

**Comment by mkoo** [comment]
This is on my docket to discuss (as Angie knows). I can reach out to the
project manager and see what sort of meeting makes sense.

Also not limited to cultural collections!
